### PR TITLE
WIP : Fix terms of use component

### PR DESCRIPTION
**Pull request by Sinisa198**

### CODE REVIEW

File: src/components/Header-Termsofuse/HeaderTermsofuse.jsx [hunk 1]
```diff
@@ -1,40 +1,41 @@
...
+              </li>
+              <li>
+                <p>Pomoć</p>
+              </li>
+              <li>
+                <p>Kontakt</p>
+              </li>
+            </ul>
+            <div className='ham-menu'>
+              <span className='line line1'></span>
```
Comment (koskeee) [inline src/components/Header-Termsofuse/HeaderTermsofuse.jsx]: This three lines can be replaced with icon from design

File: src/pages/AdditionalInformation/FrequentlyAskedQuestions.jsx [hunk 2]
```diff
@@ -10,33 +10,32 @@ const TermsOfUse = () => {
       <div className='second-box-terms'>
         <div className='box-terms'>
```
Comment (koskeee) [inline src/pages/AdditionalInformation/FrequentlyAskedQuestions.jsx:11]: Why is file name `FrequentlyAskedQuestions.jsx`, and component inside `TermsOfUse`

File: src/pages/AdditionalInformation/FrequentlyAskedQuestions.jsx [hunk 3]
```diff
@@ -10,33 +10,32 @@ const TermsOfUse = () => {
...
           <div className='list-terms'>
-            <br />
+            <br /> <br />
```
Comment (koskeee) [inline src/pages/AdditionalInformation/FrequentlyAskedQuestions.jsx]: Why are these `<br />` tags used in here. If you want to add spaces between buttons you should have different approach

File: src/pages/AdditionalInformation/PrivacyNotices.jsx [hunk 4]
```diff
@@ -17,15 +17,15 @@ const TermsOfUse = () => {
             >
               Često postavljana pitanja
             </NavLink>
-            <br />
+            <br /> <br />
```
Comment (koskeee) [inline src/pages/AdditionalInformation/PrivacyNotices.jsx]: Use some different approach

File: src/pages/AdditionalInformation/TermsOfUse.jsx [hunk 5]
```diff
@@ -17,16 +17,15 @@ const TermsOfUse = () => {
...
             </NavLink>
-            <br />
-
+            <br /> <br />
             <NavLink
               to='/privacynotices'
               className='text-for-terms-use'
               activeClassName='active'
             >
               Obaveštenje o privatnosti
```
Comment (koskeee) [inline src/pages/AdditionalInformation/TermsOfUse.jsx]: Why do we have same menu in 3 different screens. You should use different component just for that(menu) or make some bigger part of page reusable

**Review by koskeee — CHANGES_REQUESTED** [review]
Check comments and be careful with branch name. You have typed `redesing` instead `redesign`. You dont need to change that now, but just be careful

File: src/components/Header-Termsofuse/HeaderTermsofuse.jsx (hunk 1 above)
Comment (Sinisa198) [reply src/components/Header-Termsofuse/HeaderTermsofuse.jsx]: Burger menu component created and styled

File: src/pages/AdditionalInformation/FrequentlyAskedQuestions.jsx (hunk 2 above)
Comment (Sinisa198) [reply src/pages/AdditionalInformation/FrequentlyAskedQuestions.jsx:11]: The text has changed and is now in every component it belongs to

File: src/pages/AdditionalInformation/FrequentlyAskedQuestions.jsx (hunk 3 above)
Comment (Sinisa198) [reply src/pages/AdditionalInformation/FrequentlyAskedQuestions.jsx]: Deleted

File: src/pages/AdditionalInformation/PrivacyNotices.jsx (hunk 4 above)
Comment (Sinisa198) [reply src/pages/AdditionalInformation/PrivacyNotices.jsx]: Deleted

File: src/pages/AdditionalInformation/TermsOfUse.jsx (hunk 5 above)
Comment (Sinisa198) [reply src/pages/AdditionalInformation/TermsOfUse.jsx]: Deleted

File: src/App.js [hunk 6]
```diff
@@ -20,7 +19,6 @@ const App = () => {
         <Route path='/navigation' element={<Navigation />} />
         <Route path='/forms' element={<Forms />} />
         <Route path='/' element={<HomePage />} />
```
Comment (koskeee) [inline src/App.js]: Please put this HomePage route to the top of route paths

File: package.json [hunk 7]
```diff
@@ -18,6 +18,7 @@
     "match-sorter": "^6.3.1",
     "react": "^18.2.0",
     "react-axios": "^2.0.6",
+    "react-burger-menu": "^3.0.9",
```
Comment (koskeee) [inline package.json]: Please create custom burger menu. Not dependency from npm

File: src/components/BurgerMenu/BurgerMenu.jsx [hunk 8]
```diff
@@ -0,0 +1,28 @@
+import React, { useState } from 'react';
+import { slide as Menu } from 'react-burger-menu';
+import { NavLink } from 'react-router-dom';
+
+const BurgerMenu = () => {
```
Comment (koskeee) [inline src/components/BurgerMenu/BurgerMenu.jsx:4]: Please use your own burger menu

File: src/components/Footer/footer.scss [hunk 9]
```diff
@@ -1,5 +1,6 @@
 .footer-main {
   display: flex;
+  position: absolute;
```
Comment (koskeee) [inline src/components/Footer/footer.scss]: If you are using `position: absolute;`, please then use also some other params for positioning. ExampleL left, right and so

File: src/components/Header-Restaurant/Header-Restaurant.jsx [hunk 10]
```diff
@@ -5,10 +5,13 @@ import { AiTwotoneStar } from 'react-icons/ai';
 import favorite from '../../images/input-icon-first.svg';
 import arrow from '../../images/input-second-icon.svg';
 import timerIcon from '../../images/timer-for-closed.png';
+import BurgerMenu from '../BurgerMenu/BurgerMenu';
 
 const HeaderRestaurant = () => {
   return (
     <div className='main-header'>
+      <BurgerMenu />
```
Comment (koskeee) [inline src/components/Header-Restaurant/Header-Restaurant.jsx]: Already said before

File: src/pages/AdditionalInformation/FrequentlyAskedQuestions.jsx (hunk 2 above)
Comment (koskeee) [reply src/pages/AdditionalInformation/FrequentlyAskedQuestions.jsx:11]: I see here that nothing change, files `FrequentlyAskedQuestions` and `PrivacyNotes` still have same component TermsOfUse. Have you changed that in another branch or?

File: src/pages/AdditionalInformation/PrivacyNotices.jsx [hunk 11]
```diff
@@ -11,29 +11,26 @@ const TermsOfUse = () => {
...
-            <br />
             <NavLink
               to='/privacynotices'
               activeClassName='active'
               className='text-for-terms-use'
             >
               Obaveštenje o privatnosti
             </NavLink>
-            <br />
             <NavLink
```
Comment (koskeee) [inline src/pages/AdditionalInformation/PrivacyNotices.jsx:27]: You need to align properly these `NavLink` components. See this in app
Screenshot 2023-01-24 at 11 41 00

**Review by koskeee — CHANGES_REQUESTED** [review]
Check comments

File: src/components/Header-Termsofuse/HeaderTermsofuse.jsx (hunk 1 above)
Comment (koskeee) [reply src/components/Header-Termsofuse/HeaderTermsofuse.jsx]: Not working properly, please use custom component.

File: src/pages/AdditionalInformation/TermsOfUse.jsx (hunk 5 above)
Comment (koskeee) [reply src/pages/AdditionalInformation/TermsOfUse.jsx]: In another PR, or?

File: src/pages/AdditionalInformation/FrequentlyAskedQuestions.jsx (hunk 2 above)
Comment (Sinisa198) [reply src/pages/AdditionalInformation/FrequentlyAskedQuestions.jsx:11]: Changes are found on the fixed impressions component

File: src/pages/AdditionalInformation/PrivacyNotices.jsx (hunk 4 above)
Comment (Sinisa198) [reply src/pages/AdditionalInformation/PrivacyNotices.jsx]: Changes are found on the fixed impressions component

File: src/pages/AdditionalInformation/TermsOfUse.jsx (hunk 5 above)
Comment (Sinisa198) [reply src/pages/AdditionalInformation/TermsOfUse.jsx]: Changes are found on the fixed impressions component

File: src/pages/AdditionalInformation/PrivacyNotices.jsx (hunk 11 above)
Comment (Sinisa198) [reply src/pages/AdditionalInformation/PrivacyNotices.jsx:27]: Changes are found on the fixed impressions component

File: src/components/Header-Termsofuse/HeaderTermsofuse.jsx (hunk 1 above)
Comment (Sinisa198) [reply src/components/Header-Termsofuse/HeaderTermsofuse.jsx]: Create my custom component

File: src/App.js (hunk 6 above)
Comment (Sinisa198) [reply src/App.js]: Put Route HomePage on top

File: package.json (hunk 7 above)
Comment (Sinisa198) [reply package.json]: Deleted react-burger-menu: 3.0.9 and create my custom component

File: src/components/BurgerMenu/BurgerMenu.jsx (hunk 8 above)
Comment (Sinisa198) [reply src/components/BurgerMenu/BurgerMenu.jsx:4]: Deleted, and create own burger menu

Comment (koskeee) [reply src/components/BurgerMenu/BurgerMenu.jsx:4]: Burger menu is  still not shown well, as you see in screenshot i provided. Also if user clicks outside of burger menu, menu should collapse. Here burger menu collapses only if you click on  some option. 

Screenshot 2023-01-25 at 14 15 24

File: src/components/Header-Termsofuse/HeaderTermsofuse.jsx [hunk 12]
```diff
@@ -1,55 +1,32 @@
...
-            </ul>
-            <div className='ham-menu'>
-              <span className='line line1'></span>
-              <span className='line line2'></span>
-              <span className='line line3'></span>
-            </div>
-          </div>
-          <img src={termsofuseLogo} alt='' className='logo-termsofuse' />
-        </div>
+        <BurgerMenu />
```
Comment (koskeee) [inline src/components/Header-Termsofuse/HeaderTermsofuse.jsx:12]: Same as I wrote for restaurants

**Review by koskeee — CHANGES_REQUESTED** [review]
Please checkup comments

**Pull request closed without merging**